hack/make-rules/test-integration.sh

```diff
@@ -90,8 +90,17 @@ if [[ -n "${KUBE_TEST_ARGS}" ]]; then
   runTests v1
 fi
 
+# Pass arguments that with "-" and move them to goflags.
+what_flags=""
+for arg in "$@"; do
+  if [[ "${arg}" == -* ]]; then
+    what_flags="${what_flags} \"${arg}\""
```

Comment:
> what_flags=()
> for arg in "$@"; do
>   if [[ "${arg}" == -* ]]; then
>     what_flags+=("${arg}")
>   fi
> done
> 
> echo "${what_flags[@]}"

Didn't work.  It dropped the second flag, so `hack/test-integration.sh auth -test.run=TestKindAuthorization -v` didn't give me the `-v`.